### PR TITLE
Making compatible with PHP 7.4 and 8.0.

### PR DESCRIPTION
**Pull request by brandonsavage**
These changes make Aura.Auth compatible with PHP 7.4 and 8.0.

Changes:
* Removing unneeded ContainerTest.php
* Removing unneeded Aura.Di, which ContainerTest depended upon.
* Upgrading PHPUnit to run on PHP 8 and on PHP 7.2, which is the minimum version for Aura.Auth 4.x.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,7 +16,8 @@
         }
     ],
     "require": {
-        "php": ">=7.2.0"
+        "php": ">=7.2.0",
+        "vimeo/psalm": "4.x-dev"
```
Comment (harikt) [inline composer.json]: can we move this to require-dev ?

**Review by harikt — COMMENTED** [review]
I am ok with the changes. 👍

File: composer.json (hunk 1 above)
Comment (brandonsavage) [reply composer.json]: This was moved. Please take another quick look.

**Review by cxj — APPROVED** [review]
Looks good.  Thanks for all the effort!

**Pull request merged**